botocore/session.py

```diff
@@ -53,7 +49,9 @@ class Session(object):
     :ivar profile: The current profile.
     """
 
-    AllEvents = {
+    #: A dictionary where each key is an event name and the value
+    #: is the formatting string used to construct a new event.
+    ALL_EVENTS = {
```

Comment:
I do not even sure what they are used for as they are inaccurate. If the functionality is not really used anymore, I would be in favor of removing the global dictionary and any methods that use the dictionary.